src/Components/News/NewsList.css

```diff
@@ -0,0 +1,6 @@
+.card-columns {
```

Comment:
I don't think this is a good idea because `.card-columns` is a bootstrap class with their own behaviour and here we're overwriting that behaviour and I think we can do better and we have 2 options here:

1. We use the `CardColumns` but we pass a different className
2. We don't use the `CardColumns` and just add a normal `<div className="news-list">` and in `.news-list` we do the css grid thing.